src/errors.rs

```diff
@@ -17,11 +16,11 @@ pub enum Error<E> {
     ///
     /// Typically this indicates a faulty implementation of `serde::Serialize` or
     /// `serde::Deserialize`.
-    ClientDeserialize(bincode::serde::DeserializeError),
+    ClientDeserialize(::bincode::Error),
```

Comment:
Oh, maybe we should coalesce the server serialization errors as well, for consistency?